src/js/edu-benefits/1990e/config/form.js

```diff
@@ -31,7 +48,36 @@ const formConfig = {
       pages: {
         applicantInformation: {
           path: 'applicant-information',
-          title: 'Applicant information'
+          title: 'Applicant information',
+          initialData: {},
+          uiSchema: {
+            applicantFullName: fullName.uiSchema,
+            applicantSocialSecurityNumber: ssn.uiSchema,
+            applicantDateOfBirth: currentOrPastDate.uiSchema('Applicant date of birth'),
```

Comment:
The outline I'm using is pretty rough and looks like the language will need to be refined: https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Education%20Benefits/1990E/1990E-outline-draft.md
E.g., I'm guessing DOB is too informal, but I'm just guessing on that and other fields.
Let me know if there's a better source for exact language.